packages/components/src/components/domainproperties/issues/IssuesPropertiesPanel.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+    InjectedDomainPropertiesPanelCollapseProps,
+    withDomainPropertiesPanelCollapse,
+} from '../DomainPropertiesPanelCollapse';
+import {getCoreGroups, getCoreUsersInGroups} from '../../permissions/actions';
+import { Principal } from '../../..';
+
+import { AssignmentOptions, IssuesBasicPropertiesFields } from './IssuesPropertiesPanelFormElements';
+import { IssuesModel } from './models';
+import {UserGroup} from "../../permissions/models";
+
+const PROPERTIES_HEADER_ID = 'issues-properties-hdr';
+
+interface OwnProps {
+    model: IssuesModel;
+    onChange: (model: IssuesModel) => void;
+    successBsStyle?: string;
+}
+
+type Props = OwnProps & BasePropertiesPanelProps;
+
+interface State {
+    isValid: boolean;
+    coreGroups: List<Principal>;
+    coreUsers: List<UserGroup>;
+}
+
+export class IssuesPropertiesPanelImpl extends React.PureComponent<
+    Props & InjectedDomainPropertiesPanelCollapseProps,
+    State
+> {
+    constructor(props: Props & InjectedDomainPropertiesPanelCollapseProps) {
+        super(props);
+
+        this.state = {
+            isValid: true,
+            coreGroups: List<Principal>(),
+            coreUsers: List<UserGroup>()
+        };
+    }
+
+    componentDidMount() {
+        getCoreGroups().then((coreGroupsData: List<Principal>) => {
+            this.setState(() => ({
+                coreGroups: coreGroupsData,
+            }));
+        });
+
+        getCoreUsersInGroups().then((coreUsersData: List<UserGroup>) => {
+            this.setState(() => ({
+                coreUsers: coreUsersData,
+            }));
+        });
+    }
+
+    updateValidStatus = (newModel?: IssuesModel) => {
+        const { model, onChange } = this.props;
+        const updatedModel = newModel || model;
+        const isValid = updatedModel && updatedModel.hasValidProperties();
+        this.setState(
+            () => ({ isValid }),
+            () => {
+                if (newModel) {
+                    onChange(updatedModel);
+                }
+            }
+        );
+    };
+
+    onInputChange = e => {
+        const id = e.target.id;
+        let value = e.target.value;
+
+        // special case for empty string, set as null instead
+        if (Utils.isString(value) && value.length === 0) {
+            value = null;
+        }
+
+        this.onChange(id, value);
+    };
+
+    onChange = (identifier, value): void => {
+        const { model } = this.props;
+
+        // Name must be set on Domain as well
+        let newDomain = model.domain;
+        if (identifier == 'name') {
+            newDomain = model.domain.merge({ name: value }) as DomainDesign;
+        }
+
+        const newModel = model.merge({
+            [identifier]: value,
+            domain: newDomain,
+        }) as IssuesModel;
+
+        this.updateValidStatus(newModel);
+    };
+
+    onSelectChange = principal => {
+        this.onChange('assignedToGroup', principal.userId);
```

Comment:
aren't there two select inputs in the Assignment Options section? I assume this function will be called for select input change for both the "Assigned to list comes from" and the "Default user assignment". Looks like this is hardcoded to just `assignedToGroup` but should also handle `assignedToUser`, right?

Reply:
now fixed